Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -881,23 +889,23 @@ void GameList::keyPressEvent(QKeyEvent* event)
 
 void GameList::OnColumnVisibilityToggled(const QString& row, bool visible)
 {
-  static const QMap<QString, int> rowname_to_col_index = {
-      {tr("Platform"), GameListModel::COL_PLATFORM},
-      {tr("Banner"), GameListModel::COL_BANNER},
-      {tr("Title"), GameListModel::COL_TITLE},
-      {tr("Description"), GameListModel::COL_DESCRIPTION},
-      {tr("Maker"), GameListModel::COL_MAKER},
-      {tr("File Name"), GameListModel::COL_FILE_NAME},
-      {tr("File Path"), GameListModel::COL_FILE_PATH},
-      {tr("Game ID"), GameListModel::COL_ID},
-      {tr("Region"), GameListModel::COL_COUNTRY},
-      {tr("File Size"), GameListModel::COL_SIZE},
-      {tr("File Format"), GameListModel::COL_FILE_FORMAT},
-      {tr("Block Size"), GameListModel::COL_BLOCK_SIZE},
-      {tr("Compression"), GameListModel::COL_COMPRESSION},
-      {tr("Tags"), GameListModel::COL_TAGS}};
-
-  m_list->setColumnHidden(rowname_to_col_index[row], !visible);
+  using column = GameListModel::Column;
+  static const QMap<QString, column> rowname_to_column = {{tr("Platform"), column::Platform},
+                                                          {tr("Banner"), column::Banner},
+                                                          {tr("Title"), column::Title},
+                                                          {tr("Description"), column::Description},
+                                                          {tr("Maker"), column::Maker},
+                                                          {tr("File Name"), column::FileName},
+                                                          {tr("File Path"), column::FilePath},
+                                                          {tr("Game ID"), column::ID},
+                                                          {tr("Region"), column::Country},
+                                                          {tr("File Size"), column::Size},
+                                                          {tr("File Format"), column::FileFormat},
+                                                          {tr("Block Size"), column::BlockSize},
+                                                          {tr("Compression"), column::Compression},
+                                                          {tr("Tags"), column::Tags}};
```

Comment:
Adding a comma after the last entry in the map should make this format a little better automatically.